README.md

```diff
@@ -55,6 +55,7 @@
 - **Use data models with associations.**
 - **Use customize model methods**
 - **Use validation for models**
+- **Use controllers to handle requests and render empty views**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly ensure that the **Acknowledge** Section in your README file is filled with information and not empty.

Reply:
@andersonlebon thanks for the review. Already updated the Acknowledge part.